extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/connection/ModeledConnection.java

```diff
@@ -157,6 +158,12 @@ public void setConfiguration(GuacamoleConfiguration config) {
     }
 
     @Override
+    public Set<String> getSharingProfileIdentifiers()
```

Comment:
No, it's just not implemented. These are the API changes only, with the actual implementation of that API coming later.

I figured that if I held off on the review of the API until the underlying implementation was complete, that would distract from the review of the API design itself, and make the PR way too large.